src/runtime_src/core/pcie/emulation/common_em/config.h

```diff
@@ -143,7 +143,8 @@ namespace xclemulation{
   enum DEBUG_MODE {
```

Comment:
These enums (DEBUG_MODE and ERTMODE) are very dangerous.    You should consider changing them to enum classes with lower case enumerators that are scoped and don't run the risk of conflicting with #defines.  
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Renum-class
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Renum-caps

Reply:
Hi Soren, can we handle this cleanup for next time. We are in SRCB, did not want to do these changes at this point of time. I will clean up it in HEAD once XRT 2021.1 branches out.